tests/components/tasmota/test_sensor.py

```diff
@@ -377,6 +380,64 @@ async def test_indexed_sensor_attributes(hass, mqtt_mock, setup_tasmota):
     assert state.attributes.get("unit_of_measurement") == "ppm"
 
 
+@pytest.mark.parametrize("status_sensor_disabled", [False])
+async def test_enable_status_sensor(hass, mqtt_mock, setup_tasmota):
+    """Test enabling status sensor."""
+    entity_reg = await hass.helpers.entity_registry.async_get_registry()
+
+    config = copy.deepcopy(DEFAULT_CONFIG)
+    mac = config["mac"]
+
+    async_fire_mqtt_message(
+        hass,
+        f"{DEFAULT_PREFIX}/{mac}/config",
+        json.dumps(config),
+    )
+    await hass.async_block_till_done()
+    await hass.async_block_till_done()
+
+    state = hass.states.get("sensor.tasmota_status")
+    assert state is None
+    entry = entity_reg.async_get("sensor.tasmota_status")
+    assert entry.disabled
+    assert entry.disabled_by == "integration"
+
+    # Enable the status sensor
+    updated_entry = entity_reg.async_update_entity(
+        "sensor.tasmota_status", disabled_by=None
+    )
+    assert updated_entry != entry
+    assert updated_entry.disabled is False
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)